cmd/fyne/internal/util/mobile.go

```diff
@@ -1,8 +1,57 @@
 package util
 
-import "strings"
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// AndroidBuildToolsPath tries to find the location of the "build-tools" directory.
+// This depends on ANDROID_HOME variable, so you should call RequireAndroidSDK() first.
+func AndroidBuildToolsPath() string {
+	env := os.Getenv("ANDROID_HOME")
+	dir := filepath.Join(env, "build-tools")
+
+	// this may contain a version subdir
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return dir
+	}
+
+	childDir := ""
+	for _, f := range files {
+		if f.Name() == "zipalign" { // no version subdir
+			return dir
+		}
+		if f.IsDir() && childDir == "" {
+			childDir = f.Name()
+		}
+	}
+
+	if childDir == "" {
+		return dir
+	}
+	return filepath.Join(dir, childDir)
+}
+
+// IsAndroid returns true if the given os parameter represents one of the Android targets.
+func IsAndroid(os string) bool {
+	return strings.Index(os, "android") == 0
```

Comment:
This uses `strings.Index`, whereas the function below uses `strings.HasPrefix` - is there an advantage to one over the other?
Could they be unified, where `IsMobile` returns `os == "ios" || IsAndroid(os)`